blotztask-api/Models/TaskItemDTO.cs

```diff
@@ -9,5 +11,6 @@ public class TaskItemDTO
         public bool IsDone { get; set; }
         public DateTime CreatedAt { get; set; }
         public DateTime UpdatedAt { get; set; }
+        public Label Label { get; set; }
```

Comment:
instead of using label entity, i will create a new DTO class for label

```suggestion
        public Label Label { get; set; }
        public LabelDTO Label { get; set; }

```

Reply:
Done